### PR TITLE
Add fields to `PlattformBlockedFaultConfiguration`

### LINKED ISSUE #105: Add `PlatformBlockedFaultConfiguration` class
## User stories
As a user, that wants to configure a `PlatformBlockedFault` I want to know the configuration fields the  in order to add it.

## (Wiki) References
- https://github.com/BP2022-AP1/bp2022-ap1/wiki/Architecture#database-class-diagram

## Acceptance Criteria
- [ ] Add python class with peewee and marshmellow, without migrations and foreign keys
- [ ] Update database class diagram in wiki

### PR DESCRIPTION
**Pull request by instant-sky**
Fixes #105 

Description (what might a PO or reviewer want to know?)

## PR checklist

- [ ] Acceptance criteria fulfilled
- [ ] Additional features are tested
- [ ] Docs (code, wiki & diagrams) updated
- [ ] Breaking changes anounced
- [ ] Dev-branch has been merged into local branch to resolve conflicts
- [ ] Tests and linter have passed AFTER local merge
- [ ] Another dev reviewed and approved

### CODE REVIEW

File: src/fault_injector/fault_types/fault.py [hunk 1]
```diff
@@ -47,21 +47,21 @@ def next_tick(self, tick: int):
...
+        start_tick = marsh.fields.Integer(required=True)
+        end_tick = marsh.fields.Integer(required=True)
         description = marsh.fields.String()
 
         def _make(self, data: dict) -> "FaultConfiguration":
             return FaultConfiguration(**data)
 
-    start_tick = BigIntegerField()
-    end_tick = BigIntegerField()
+    start_tick = BigIntegerField(null=False)
```
Comment (CR1337) [inline src/fault_injector/fault_types/fault.py]: Is a normal IntegerField too small?

**Review by CR1337 — APPROVED** [review]
Looks good. Approved :+1:

File: poetry.lock [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# This file is automatically @generated by Poetry and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.4.2 and should not be changed by hand.
```
Comment (antonykamp) [inline poetry.lock]: Whats going on here? 🤔  Why did you add the lock file to the PR?

File: tests/fault_injector/test_fault_configs.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import pytest
+
+
+class TestPlatformBlockedFaultConfiguration:
```
Comment (antonykamp) [inline tests/fault_injector/test_fault_configs.py]: Where do you use the fixtures? 🤔

**Review by antonykamp — COMMENTED** [review]
Great PR 👍 We're making progress 🙌 
Please take a small look into your defined fixtures again and in the lock-file 😄

File: src/fault_injector/fault_types/fault.py (hunk 1 above)
Comment (instant-sky) [reply src/fault_injector/fault_types/fault.py]: Depends on how long the simulations are supposed to run. Probably a normal IntegerField is enough...

File: poetry.lock (hunk 2 above)
Comment (instant-sky) [reply poetry.lock]: Good question, I had to use poetry install but some packages were out of date. I don't really know, why the version is statet now explicitly...

File: tests/fault_injector/test_fault_configs.py (hunk 3 above)
Comment (instant-sky) [reply tests/fault_injector/test_fault_configs.py]: Ah right, these are obsolete now. Thank you for mentioning it!

**Review by antonykamp — APPROVED** [review]
